src/main/java/frc/robot/subsystems/ColorOptions.java

```diff
@@ -0,0 +1,5 @@
+package frc.robot.subsystems;
```

Comment:
Probably not for the subsystem package.  Is there a more central place for constants?

Reply:
Yes, moved it to Utils.